### PR TITLE
Create CC audiology special case page

### PR DESCRIPTION
**Pull request by narin**
## Description
As a veteran who's selected audiology at Community Care facility, I want to be able to specify whether I need a hearing aid or audiology appointment.

## Testing done
Tested locally and added unit test

## Screenshots
image

## AC:
- [ ] Page shows for veterans who've previously selected CC and audiology care type

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/containers/TypeOfAudiologyCarePage.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import { getFormPageInfo } from '../utils/selectors';
+
+const initialSchema = {
+  type: 'object',
+  required: ['ccId'],
+  properties: {
+    ccId: {
```
Comment (jbalboni) [inline src/applications/vaos/containers/TypeOfAudiologyCarePage.jsx]: Might make more sense to call this `audiologyType`? Or if we think we might use it for other things, `secondaryTypeOfCareId`?

File: src/applications/vaos/sass/vaos.scss [hunk 2]
```diff
@@ -42,3 +42,13 @@
     margin-top: 0.6em;
   }
 }
+
+.vaos-form__audiology-type {
```
Comment (jbalboni) [inline src/applications/vaos/sass/vaos.scss]: Could we combine this with the `.vaos-form__appt-type` styles? It seems like the same kind of UI.

**Review by jbalboni — APPROVED** [review]
Couple things, but looks good otherwise.

File: src/applications/vaos/containers/TypeOfAudiologyCarePage.jsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+            <span className="vads-u-display--block vads-u-font-size--lg vads-u-font-weight--bold">
+              Routine hearing exam
+            </span>
+            <span className="vads-u-display--block vads-u-font-size--sm">
+              This incudes an office visit for a hearing exam and an evaluation
+              using non-invasive tests to check your hearing and inner ear
+              health. A routine exam is not meant for any new or sudden changes
+              with your hearing or ears.
+            </span>
+          </>
```
Comment (cvalarida) [inline src/applications/vaos/containers/TypeOfAudiologyCarePage.jsx]: Oh no! GitHub's syntax highlighting chokes up at this! :crying_cat_face:

File: src/applications/vaos/containers/TypeOfAudiologyCarePage.jsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+  render() {
+    const { schema, data, pageChangeInProgress } = this.props;
+
+    return (
+      <div className="vaos-form__detailed-radio">
+        <h1 className="vads-u-font-size--h2">
+          Choose the type of audiology care you need
+        </h1>
+        <SchemaForm
```
Comment (cvalarida) [inline src/applications/vaos/containers/TypeOfAudiologyCarePage.jsx]: Oh, interesting. So you're using `SchemaForm` to generate "form pages" without using our full-blown form system.

Cool. :+1:

**Review by cvalarida — APPROVED** [review]
Cool! I really like pulling out just what you need from the forms system. :100:

**Pull request merged**